bin/check-nvmrc.js

```diff
@@ -17,17 +17,25 @@ fs.readFile(path.join(__dirname, '../.nvmrc'), 'utf8', function (error, data) {
     process.exit()
   }
 
+  var nvmInstallText = 'To do this you can install nvm (https://github.com/creationix/nvm) then run `nvm install`.'
+
   if (versionMatchesMajor) {
     console.log('' +
       'Warning: You are using Node.js version ' + currentVersion + ' which we do not use. ' +
+      '\n\n' +
```

Comment:
wouldn't have to be done in this PR (non blocking) but would this be clearer using backticks? `

Reply:
This code needs to be written in a way that'll run in older versions of Node.js, backticks are only avaliable in newer versions of Node.js